tests/OptionTests.cs

```diff
@@ -97,6 +97,13 @@ public void ToOptionWithNullableNull()
             Assert.That(result, Is.EqualTo(Option.None<int>()));
         }
 
+        [Test]
+        public void FromTuple()
+        {
+            var result = Option.From((true, 42));
```

Comment:
What about a test for when the first item of the tuple is `false`, as in `Option.From((false, …))`?